packages/reliability-tests/src/run-tests.js

```diff
@@ -3,7 +3,7 @@ import { promisify } from "util";
 
 const execAsync = promisify(exec);
 
-const WAKUNODE_IMAGE = process.env.WAKUNODE_IMAGE || "wakuorg/nwaku:v0.35.1";
+const WAKUNODE_IMAGE = process.env.WAKUNODE_IMAGE || "wakuorg/nwaku:v0.36.0";
```

Comment:
I would suggest you do the upgrade in a separate PR. Just mount light push v2 if needed.

Reply:
100%